### ISSUE TITLE
[Enhancement/Question] Making a complete pack for vanilla blocks/items?

### ISSUE THREAD

**Issue by AestheticGoat** [ianthe-one/porcelain-mask#1]
Hey! I see your mod hasn't gotten very much attention yet, but I think these animations done in the example pack are stellar! Honestly, if you made a mod or resource pack that just does what the example does, but for every item and block, that would be amazing! I would love to see this be pursued! Anywho, just a suggestion. I think your mod is fantastic and has a whole lot of potential. Keep up the good work!

**Comment by ianaduarte** [comment]
I'll look into it but it feels a little out of scope.

**Comment by VicoIlFico** [comment]
I agree with op, it could become a very popular mod if if it added full animations in third person, they look absolutely amazing with the first person model mod too! Once it gains a little bit of popularity I suggest making a discord server to host a community where packs can be created and shared by users using the mod's library
